utilities/bash/run_six.sh

```diff
@@ -2301,6 +2357,13 @@ if ${lsubmit} ; then
 	sed -i "s#^queue dirname from.*#queue dirname from ${sixdeskjobs}/${LHCDesName}.list#g" ${sixdeskjobs}/htcondor_run_six.sub
 	sed -i "s#^+JobFlavour =.*#+JobFlavour = \"${HTCq}\"#g" ${sixdeskjobs}/htcondor_run_six.sub
     fi
+
+    if [ "$sixdeskplatform" == "htboinc" ] ; then
```

Comment:
here we should modify the .sub in such a way that not the dir, but the whole path to the .zip is dealt with